hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/event/Event.java

```diff
@@ -48,13 +51,16 @@
 
     // A description of the event, suitable for display
     @JsonInclude
-    private String eventText;
+    private String text;
 
     @JsonInclude(Include.NON_EMPTY)
     private Map<String, String> context;
 
+    // TODO: WE NEED TO POTENTIALLY CONSOLIDATE OUR TAG APPROACH.  I THINK MAYBE WE SHOULD MOVE THE CURRENT Tag TO
+    // A NAME-VALUE PAIR, GET RID OF HIDDEN, AND CHANGE triggerId to a more generic ID.  The Tag as is is not
+    // going to work here, but for now leave as is/.
     @JsonInclude(Include.NON_EMPTY)
-    private Map<String, String> tags;
+    private Set<Tag> tags;
```

Comment:
I am agree with the comment.
Perhaps the Tag model works fine but Trigger but I think we should study a lightweight approach for Events, or even for Triggers too.
I must say I liked the Map.tags, it looked intuitive coming from a REST approach.
I don't know perhaps we can talk about tags also to see which effort could be more interesting to consolidate.